static/gsApp/views/amCheckout/steps/planSelectRow.tsx

```diff
@@ -84,11 +91,15 @@ function PlanSelectRow({
 
   const describeId = `plan-details-${plan.id}`;
   const hasFeatures = !!Object.keys(features || {}).length;
-  const errorsStartingPrice = plan.planCategories.errors
-    ? plan.planCategories.errors[1]?.onDemandPrice
+  const errorsStartingPrice = shouldShowEventPrice
+    ? plan.planCategories.errors
+      ? plan.planCategories.errors[1]?.onDemandPrice
+      : null
```

Comment:
LGTM
nit: nested ternary can be hard to read.
Pro tip: There is an [Array at](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#description) method to get an array element.
```js
plan.planCategories.errors?.at(1)?.onDemandPrice || null
```

Reply:
can address this in a follow up as i'd like to clear this from the UI asap, thanks!

Reply:
No worries. No action needed, just FYI.